src/tokens/bsc.json

```diff
@@ -8,6 +8,30 @@
       "image": "ir.svg",
       "name": "Infrared Governance Token",
       "symbol": "IR"
+    },
+    {
+      "address": "0xbb4cdb9cbd36b01bd1cbaebf2de08d9173bc095c",
+      "categories": ["project"],
+      "chainId": 56,
+      "decimals": 18,
+      "image": "wbnb.svg",
+      "name": "Wrapped BNB",
+      "symbol": "WBNB"
+    },
+    {
+      "address": "0xd9b7a1d544432b0bacac321ccf543f3a23ae55de9709b2cf6664ab83f4cbe0a4",
```

Comment:
This doesn't seem valid? I don't see it on https://bscscan.com/tx/0xd9b7a1d544432b0bacac321ccf543f3a23ae55de9709b2cf6664ab83f4cbe0a4